__init__.py

```diff
@@ -103,7 +129,12 @@ def on_websettings_changed(self):
     # Creates dialogs for errors and speaks them
     # Returns None if nothing was found
     # Else returns entity that was found
-    def _find_entity(self, entity, domains):
+    def _find_entity(self, entity: str, domains: list(str)) -> dict:
+        """Handle communication with HA client for entity finding
+
+        Returns:
+            Entity
```

Comment:
Well beyond the scope of this PR - but this return statement made me wonder if it would be worth defining a small `Entity` class given the importance of it in this system?

It's then clearer what we are passing around, and what we can expect from it. Methods like `find_entity_attr` could also be refactored into it to better cluster code that really belongs together.

Definitely not for this PR - just musing.

Reply:
I will rewrite some functions as we started to pass Entity atributes in first search for it so thing like find entity attr and second red from WS is not needed.